openid-sharedsignals-framework-1_0.md

```diff
@@ -564,11 +535,44 @@ verification_endpoint
 
 critical_subject_members
 
-> OPTIONAL. List of member names in a Complex Subject which, if present in
+> OPTIONAL. An array of member names in a Complex Subject which, if present in
   a Subject Member in an event, MUST be interpreted by a Receiver.
+  
+supported_scopes
+
+> OPTIONAL. A list of OAuth {{RFC6749}} scope names that the Transmitter supports for specific endpoints. The value of this field is a JSON object that has the endpoint names as keys, and arrays of scope name strings they support as their values. OAuth tokens obtained using any of the scopes defined here MUST be accepted by the specified endpoint. Any key that is not defined as an endpoint in the Transmitter Configuration Metadata MUST be ignored
```

Comment:
> Any key that is not defined as an endpoint in the Transmitter Configuration Metadata MUST be ignored

Suggestion to rephrase. 

> Any endpoint key that is not defined in `supported_scopes` is assumed to not require OAuth tokens for authorization

Reply:
I added this line to the existing text because these are two different things.